committee-steering/governance/sig-governance-template-short.md

```diff
@@ -50,6 +50,13 @@ Membership for roles tracked in: <link to OWNERS file>
   - *MAY* participate in decision making for the subprojects they hold roles in
   - Includes all reviewers and approvers in [OWNERS] files for subprojects
 
+- Security Contact
+  - *MUST* be a contact point for the Product Security Team to reach out to for
+    triaging and handling of incoming issues
+  - *MUST* accept the [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy)
+  - Defined in [SECURITY_CONTACTS] files, this is only relevant to the root file in
```

Comment:
Do you want backticks instead of brackets here?  Althernatively, could you add a reference [like this][1] (which currently 404s, but that's a separate issue)?  If you add the reference, you'll want a `SECURITY_CONTACTS` in this repo (probably not a bad idea anyway).

[1]: https://github.com/kubernetes/community/blame/0a614ded6f114487dd78b5cec4329d19df8711d4/committee-steering/governance/sig-governance-template-short.md#L137